test/test_rpc_endpoints.py

```diff
@@ -0,0 +1,820 @@
+"""
+Tests RPC endpoints.
+"""
+from __future__ import annotations
+
+import json
+import typing
+
+import pytest
+from starkware.starknet.public.abi import get_storage_var_address, get_selector_from_name
+from starkware.starknet.services.api.contract_class import ContractClass
+from starkware.starknet.core.os.class_hash import compute_class_hash
+from starkware.starknet.services.api.gateway.transaction import Transaction, Deploy
+
+from starknet_devnet.server import app
+from starknet_devnet.general_config import DEFAULT_GENERAL_CONFIG
```

Comment:
The DEFAULT_GENERAL_CONFIG is still used in some places, can it be replaced with state.general_config?

Reply:
After discussing this - it makes sense to use DEFAULT_GENERAL_CONFIG directly in tests.